src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/IBackupUtilities.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.SqlTools.ServiceLayer.Admin;
+using Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts;
+using System.Data.SqlClient;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery
+{
+    public interface IBackupUtilities
```

Comment:
>IBackupUtilities [](start = 21, length = 16)

please add C# document comments for this interface and every method in the interface...our public interfaces need to be documented.